semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/EventIncubatingAttributes.java

```diff
@@ -15,34 +15,19 @@
 public final class EventIncubatingAttributes {
 
   /**
-   * The domain identifies the business context for the events.
+   * Identifies the class / type of event.
    *
    * <p>Notes:
    *
    * <ul>
-   *   <li>Events across different domains may have same {@code event.name}, yet be unrelated
-   *       events.
+   *   <li>Event names are subject to the same rules as <a
+   *       href="https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/common/attribute-naming.md">attribute
+   *       names</a>. Notably, event names are namespaced to avoid collisions and provide a clean
+   *       separation of semantics for events in separate domains like browser, mobile, and
+   *       kubernetes.
    * </ul>
    */
-  public static final AttributeKey<String> EVENT_DOMAIN = stringKey("event.domain");
```

Comment:
@lmolkova This is an example of an experimental attribute that was deleted rather retained but marked as deprecated.

Based on what you said, in the future this will not be the case. And we would expect to keep this attribute around semi-permanently marked as deprecated. Is that correct?

Reply:
I'm not trying to answer for @lmolkova, and I'm not addressing the general question being asked....but in this specific case I do think it makes sense to remove it, because (experimental) Events are starting to gel and we don't want people using this.

Reply:
sorry, I missed it.
Yes, starting from 1.24.0 we'll have a check that prevents attributes (and metrics) from being deleted - https://github.com/open-telemetry/semantic-conventions/pull/764 

It would apply to experimental and stable semconv. 

I think we'll need to do the same for events, updated https://github.com/open-telemetry/build-tools/issues/292  so we don't forget

Reply:
Thanks, I guess that's fine, as long as they're marked deprecated and can be removed in the subsequent release. I'd hate for us to have constraints that cause long-term maintenance burden due to changes in experimental semconv (and yes, I realize this isn't the place to drag this out, heh 😅 ).

Reply:
Would love to hear your thoughts on removing-experimental-deprecated on https://github.com/open-telemetry/semantic-conventions/issues/740. It'd be great to have a common policy for all language SIGs.